.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug Fix
+assignees: ''
+
+---
+
```

Comment:
It might be nice to have something like

- [ ] I have searched for this bug on the Issues page and am convinced this is not a duplicate.

Reply:
This is a template, not a checklist